CONTRIBUTING.md

```diff
@@ -177,6 +179,9 @@ These analyses follow the [Google R Style Guide](https://google.github.io/styleg
 Snakemake will automatically runs the [r-lib/styler package](https://github.com/r-lib/styler)  on each `.Rmd` file called in the `Snakefile`.
 This will help fix some spacing and formatting issues automatically.
 
+Github actions will also automatically run styler and commit back to your branch any changes.
+See the [Github actions section](#github-actions-summary) for more info on how the automation steps for this works.
```

Comment:
```suggestion
See the [Github actions section](#github-actions) for more info on how the automation steps for this works.
```

When I click here it doesn't automatically redirect to the `Github actions` section and I believe that may be because of the extra word `summary` here. This is just a guess!